ScalaScript.sublime-syntax

```diff
@@ -51,9 +62,9 @@ contexts:
       scope: invalid.illegal.stray-bracket-end
 
     # Blocks
-    - match: '\{|(?i)\bevent\b'
+    - match: '\{|(?i)\bEvent\b'
       push: block
-    - match: '\}|(?i)\bend\b'
+    - match: '\}|(?i)\bEnd\b'
```

Comment:
They're still [listed in the current docs](https://docs.scala.com/display/D1102/Lexical+Rules#LexicalRules-ControlPrimitives) as control primitives.